src/transport_ofi.c

```diff
@@ -729,8 +767,13 @@ int publish_mr_info(void)
             heap_key = fi_mr_key(shmem_transport_ofi_target_heap_mrfd);
             data_key = fi_mr_key(shmem_transport_ofi_target_data_mrfd);
         } else {
+#ifdef ENABLE_MR_ENDPOINT
+            heap_key = 1;
+            data_key = 0;
+#else
             heap_key = 1ULL;
             data_key = 0ULL;
+#endif
```

Comment:
The if and else cases look the same?

Reply:
Thanks for catching this. Should have been corrected with the previous commit.